contents/blog/the-posthog-array-1-35-0.md

```diff
@@ -0,0 +1,109 @@
+---
+date: 2022-04-25
+title: Array 1.35.0
```

Comment:
Could we starting giving these more of a headline? Something like 'Array 1.35.0: Introducting xyz'

Reply:
Aye, this is something we've done for the last few emails and would be a good add here. Will put commit something in, but happy to tweak.